Corgibytes.Freshli.Cli/Functionality/Cache.cs

```diff
@@ -0,0 +1,84 @@
+using System;
+using System.IO;
+using System.Linq;
+using Microsoft.EntityFrameworkCore;
+using ServiceStack;
+
+namespace Corgibytes.Freshli.Cli.Functionality
+{
+    [Index(nameof(Key), IsUnique = true)]
+    public class CachedProperty
+    {
+        public int Id { get; set; }
+        public string Key { get; set; }
+        public string Value { get; set; }
+    }
+
+    class CacheContext : DbContext
```

Comment:
I'd like for us to stick to one class per file where the file name matches the class name.